Kitodo/src/main/java/org/goobi/production/cli/helper/CopyProcess.java

```diff
@@ -250,6 +251,7 @@ public void evaluateOpac() {
             fillFieldsFromConfig();
 
         } catch (Exception e) {
+            logger.error("Fehler beim Einlesen des Opac-Ergebnisses ", e);
             Helper.setFehlerMeldung("Fehler beim Einlesen des Opac-Ergebnisses ", e);
             e.printStackTrace();
```

Comment:
Please remove this `e.printStackTrace()` as you changing the error reporting in a better way.